cpp-client/deephaven/client/include/private/deephaven/client/impl/update_by_operation_impl.h

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) 2016-2023 Deephaven Data Labs and Patent Pending
+ */
+#pragma once
+
+#include "deephaven/proto/table.pb.h"
+#include "deephaven/proto/table.grpc.pb.h"
+
+namespace deephaven::client::impl {
+class UpdateByOperationImpl {
+  typedef io::deephaven::proto::backplane::grpc::UpdateByRequest::UpdateByOperation GrpcUpdateByOperation;
```

Comment:
May I suggest naming this `UpdateByOperationProto` ? (shorthand for `UpdateByOperationProtoMessage`)